lib/lhm.rb

```diff
@@ -44,13 +44,34 @@ def self.change_table(table_name, options = {}, &block)
     origin = Table.parse(table_name, connection)
     invoker = Invoker.new(origin, connection)
     block.call(invoker.migrator)
-    invoker.run(options)
-
+    migration = invoker.run(options)
     true
   ensure
+    archive_table_name = nil
+    archive_table_name = migration.archive_name unless migration.nil?
```

Comment:
Can we have this as part of Invoker#run, instead of running here, as technically the migration should also cover triggers? We could also make it a separate class and call it from Invoker class